listenbrainz/webserver/static/js/jsx/profile.jsx

```diff
@@ -282,7 +282,9 @@ class RecentListens extends React.Component {
                                 </abbr>
                               </td>
                             }
-                            {(this.state.mode === "follow" || this.state.mode === "recent") && <td><a href={`/user/${listen.user_name}`}>{listen.user_name}</a></td>}
+                            {(this.state.mode === "follow" || this.state.mode === "recent") &&
```

Comment:
Hmmm. this is starting to get ugly. :(

Reply:
Agreed. I've started thinking about what needs to be refactored, after initial beta testing.
The 'mode' certainly is part of that.